docs/apis/ledgerstate.md

```diff
@@ -0,0 +1,1112 @@
+if err != nil {
+    // return error
+}
+fmt.Println("transaction inputs:")
+for _, input := range resp.Inputs {
+    fmt.Println("inputID:", input.ReferencedOutputID.Base58)
+}
+fmt.Println("transaction outputs:")
+for _, output := range resp.Outputs{
+    fmt.Println("outputID:", output.OutputID.Base58)
+    fmt.Println("output type:", output.Type)
+}
+fmt.Println("access mana pledgeID:", resp.AccessPledgeID)
+fmt.Println("consensus mana pledgeID:", resp.ConsensusPledgeID)
+```
+### Response examples
+```json
+{
+    "version": 0,
+    "timestamp": 1621889348,
+    "accessPledgeID": "DsHT39ZmwAGrKQe7F2rAjwHseUnJeY89gDPEH1FJxYdH",
+    "consensusPledgeID": "DsHT39ZmwAGrKQe7F2rAjwHseUnJeY89gDPEH1FJxYdH",
+    "inputs": [
+        {
+            "type": "UTXOInputType",
+            "referencedOutputID": {
+                "base58": "41GvDSQnd12e4nWnd2WzmdLmffruXqsE46jgeUbnB8s1QnK",
+                "transactionID": "9wr21zza46Y5QonKEHNQ6x8puA7Rbq5LAbsQZJCK1g1g",
+                "outputIndex": 0
+            }
+        }
+    ],
+    "outputs": [
+        {
+            "outputID": {
+                "base58": "6gMWUCgJDozmyLeGzW3ibGFicEq2wbhsxgAw8rUVPvn9bj5",
+                "transactionID": "HuYUAwCeexmBePNXx5rNeJX1zUvUdUUs5LvmRmWe7HCV",
+                "outputIndex": 0
+            },
+            "type": "SigLockedColoredOutputType",
+            "output": {
+                "balances": {
+                    "11111111111111111111111111111111": 1000000
+                },
+                "address": "1HrUn1jWAjrMU58LLdFhfnWBwUKVdWjP5ojp7oCL9mVWs"
+            }
+        }
+    ],
+    "unlockBlocks": [
+        {
+            "type": "SignatureUnlockBlockType",
+            "publicKey": "12vNcfgRHLSsobeqZFrjFRcVAmFQbDVniguPnEoxmkbG",
+            "signature": "4isq3qzhY4MwbSeYM2NgRn5noWAyh5rqD12ruiTQ7P89TfXNecwHZ5nbpDc4UB7md1bkfM1xYtSh18FwLqK8HAC6"
+        }
+    ],
+    "dataPayload": ""
+}
+```
+
+### Results
+|Return field | Type | Description|
+|:-----|:------|:------|
+| `version`         | uint8  | The version of the transaction essence.   |
+| `timestamp`       | int64    | The issuing time of the transaction. |
+| `accessPledgeID`  | string      | The boolean indicator if the message is solid. |
+| `consensusPledgeID`| string     | The time of solidification of a message. |
+| `inputs`          | []Input       | The number of consumers. |
+| `outputs`         | []Output    | The firs consumer of athe output. |
+| `unlockBlocks`    | []UnlockBlock      | The boolean indicator if the transaction . |
+| `dataPayload` | []byte      | The boolean indicator if the transaction . |
+
+`Input `
+|Field | Type | Description|
+|:-----|:------|:------|
+| `Type`  | string | The type of input.   |
+| `ReferencedOutputID`   | string | The output ID that is used as an input for the transaction.     |
+| `output` | Output | An output that is used as an input for the transaction |
```

Comment:
There's no `Output` in the response above.
And add a section for `ReferencedOutputID`.